src/tests/ftest/util/core_file.py

```diff
@@ -0,0 +1,420 @@
+"""
+  (C) Copyright 2022-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from collections import defaultdict
+from fnmatch import fnmatch
+import os
+import re
+from run_utils import run_local, run_remote, find_command, RunException
+
+# One of the dfuse tests intermittently creates core files which is known so make a special case
+# for that test.
+CORE_FILES_IGNORE = {'./dfuse/daos_build.py': ('./conftest')}
+
+
+class CoreFileException(Exception):
+    """Base exception for this module."""
+
+
+class CoreFileProcessing():
+    """Process core files generated by tests."""
+
+    USE_DEBUGINFO_INSTALL = True
+
+    def __init__(self, log):
+        """Initialize a CoreFileProcessing object.
+
+        Args:
+            log (logger): object configured to log messages
+        """
+        # pylint: disable=import-outside-toplevel,import-error,no-name-in-module
+        from distro_utils import detect
+
+        self.log = log
+        self.distro_info = detect()
+
```

Comment:
Let's  get the local core file path here:
```suggestion
        host = get_local_host()
        result = self.get_core_file_pattern(host)
        self.local_core_path = result[str(host)])['path']
```

Reply:
Done.